Vagrantfile.containerd2youki

```diff
@@ -3,49 +3,54 @@
 
 Vagrant.configure("2") do |config|
     config.vm.box = "generic/ubuntu2204"
-    config.vm.synced_folder '.', '/vagrant/youki', disabled: true
+    config.vm.synced_folder '.', '/vagrant/youki', disabled: false
 
     config.vm.provider "virtualbox" do |v|
       v.memory = 4096
       v.cpus = 2
     end
 
-    config.vm.provision "shell", inline: <<-SHELL
-      set -e -u -o pipefail
-      apt-get update && apt-get install -y \
-        make \
-        pkg-config         \
-        libsystemd-dev     \
-        libdbus-glib-1-dev \
-        build-essential    \
-        libelf-dev \
-        libseccomp-dev \
-        libbtrfs-dev \
-        btrfs-progs
-      wget --quiet https://go.dev/dl/go1.18.1.linux-amd64.tar.gz -O /tmp/go1.18.1.linux-amd64.tar.gz
-      rm -rf /usr/local/go && tar -C /usr/local -xzf /tmp/go1.18.1.linux-amd64.tar.gz
-      curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
-      echo "export PATH=$PATH:$HOME/.cargo/bin:/usr/local/go/bin" >> ~/.bashrc
-      echo "export GOPATH=$HOME/go" >> ~/.bashrc
-      export PATH=$PATH:$HOME/.cargo/bin:/usr/local/go/bin
-      export GOPATH=$HOME/go
+    config.vm.provision "bootstrap", type: "shell" do |s|
+      s.inline = <<-SHELL
+        set -e -u -o pipefail
+        apt-get update && apt-get install -y \
```

Comment:
The apt-packages / go / contianerd / cni /critools will be downloaded and installed for every single run of `make containerd-test`.

IMO it would make the test runs faster if a custom vagrant box was built (and possibly uploaded to the vagrant cloud) by a separate make target say `build vagrant`. The box has to be rebuilt only when there a change in the packages / base linux image is required.

This will make code -> test -> code process faster as only the rust code will be copied & rebuilt.

Reply:
Thanks for your review :bow: 

No, these install commands only run first.
> The apt-packages / go / contianerd / cni /critools will be downloaded and installed for every single run of make containerd-test.